.github/workflows/release.yaml

```diff
@@ -465,104 +465,48 @@ jobs:
       - name: Download data
         run: sh download_data.sh
 
-      - name: Install s3cmd
+      - name: Install s3cmd (staging)
         if: github.ref != 'refs/heads/release'
         uses: ./.ci/install-s3cmd
         with:
-          access-key: ${{ secrets.STAGING_ASSETS_BUCKET_ACCESS_KEY }}
-          secret-key: ${{ secrets.STAGING_ASSETS_BUCKET_SECRET_KEY }}
+          access-key: ${{ secrets.AI_ASSETS_STG_BUCKET_ACCESS_KEY }}
+          secret-key: ${{ secrets.AI_ASSETS_STG_BUCKET_SECRET_KEY }}
+
+      - name: Install s3cmd (production)
+        if: github.ref = 'refs/heads/release'
+        uses: ./.ci/install-s3cmd
+        with:
+          access-key: ${{ secrets.AI_ASSETS_PROD_BUCKET_ACCESS_KEY }}
+          secret-key: ${{ secrets.AI_ASSETS_PROD_BUCKET_SECRET_KEY }}
 
       - name: Upload assets to S3
-        if: github.ref != 'refs/heads/release'
+        id: s3-upload
         shell: bash
-        env:
-          BUCKET_URL: s3://xayn_ai_staging_assets
         run: |
           set -e
 
+          if [[ "${{ github.ref }}" = "refs/heads/release" ]]
+            then
```

Comment:
```suggestion
          then
```

Reply:
🤔 I've always indented if-clauses in bash like that:
```bash
if condition
  then
    (...)
  else
    (...)
fi
```

Reply:
ah ok, not sure if there is any convention, i'm fine with this as well, but then the `else` needs to be indented.

Reply:
As it is already, right? Or you think it should be indented one step further still?

Reply:
I don't think there is a convention.

What I see most often is:

```
if condition ; then
   code
else
   code
fi
```

Note that the `;` is very important :wink: 

But IMHO it doesn't matter either way.

Reply:
Yeah, I'll stick with what I went with as I think it gives some good visibility, if it's really ok. Let me merge this and get it out of your way, guys! :D